controllers/openstackdataplanenodeset_controller.go

```diff
@@ -489,6 +583,10 @@ func (r *OpenStackDataPlaneNodeSetReconciler) SetupWithManager(mgr ctrl.Manager)
 			dnsMasqWatcher).
 		Watches(&dataplanev1.OpenStackDataPlaneDeployment{},
 			deploymentWatcher).
+		Watches(&corev1.ConfigMap{},
```

Comment:
This could be expensive as we watch for all configmaps/secrets, though we filter nodes using the index we create. This seems to be the pattern used else where and I don't have a better solution.